struct_test.go

```diff
@@ -0,0 +1,72 @@
+package gofakeit
```

Comment:
Can you add a couple of examples for the docs please?

Reply:
Done